### DISCUSSION TITLE
control , information displays, Text with label

### DISCUSSION THREAD

**Discussion by runtimevic** [flet-dev/flet#2183] [Q&A]
Hello,

How can I have a Text control that has a label? Just like the TextField control has, this label would be to indicate information about what the Text is displaying..
thanks you, Víctor

**Comment by ndonkoHenri** [comment]
All flet controls have a `data` property you can use to store `Any`(all types) information.

**Comment by runtimevic** [comment]
> All flet controls have a `data` property you can use to store `Any`(all types) information.

Hello, please can you give an example...

**Comment by ndonkoHenri** [comment]
https://flet.dev/docs/controls/elevatedbutton#elevated-button-with-click-event

**Comment by runtimevic** [comment]
Hello @ndonkoHenri ,
This doesn't work for me, what I would like is to have a Text control in which I could put text at the top (label) as there is in the TextField control and below the text that I want...
<img width="1217" alt="Flet_control_TextField" src="https://github.com/flet-dev/flet/assets/53791834/1eaa06df-6c5f-49ad-8419-e9b27b513e77">

<img width="1230" alt="Flet_control_Text" src="https://github.com/flet-dev/flet/assets/53791834/8226e5fa-8033-438b-9354-3206a2b591c4">

**Comment by ndonkoHenri** [comment]
Ah! It is not directly possible. You will probably have to build a custom control with the available ones.

**Comment by runtimevic** [comment]
> Ah! It is not directly possible. You will probably have to build a custom control with the available ones.

custom control how?...

**Comment by ndonkoHenri** [comment]
You will have to combine the available controls (see [docs](https://flet.dev/docs/)) inorder to come up with yours.
